packages/manager-react-components/src/components/datagrid/visibility/visibility-management.component.tsx

```diff
@@ -62,7 +79,7 @@ export function VisibilityManagement({
       >
         <div className="flex flex-col">
           {columnsVisibility
-            .filter(({ label }) => label !== '')
+            .filter(({ label }) => typeof label === 'string' && label !== '')
```

Comment:
So it means only labels typed as strings can be selected/unselected ? Is this limitation necessary ? Like if I put an icon as a column header why wouldn't I be able to show or hide this column